pkg/kubectl/resource/builder.go

```diff
@@ -120,29 +118,30 @@ func (b *Builder) URL(urls ...*url.URL) *Builder {
 	return b
 }
 
-// Stdin will read objects from the standard input. If ContinueOnError() is set
-// prior to this method being called, objects in the stream that are unrecognized
-// will be ignored (but logged at V(2)).
-func (b *Builder) Stdin() *Builder {
-	return b.Stream(os.Stdin, "STDIN")
-}
-
 // Stream will read objects from the provided reader, and if an error occurs will
 // include the name string in the error message. If ContinueOnError() is set
 // prior to this method being called, objects in the stream that are unrecognized
 // will be ignored (but logged at V(2)).
 func (b *Builder) Stream(r io.Reader, name string) *Builder {
 	b.stream = true
-	b.paths = append(b.paths, NewStreamVisitor(r, b.mapper, b.schema, name, b.continueOnError))
+	b.paths = append(b.paths, &StreamVisitor{r, b.mapper, name, b.continueOnError, b.schema})
```

Comment:
We try not to do this in general because it's fragile to refactoring.  It would be better to switch this the named struct form here if you aren't calling a method.